common/achievement/util.ts

```diff
@@ -19,8 +21,18 @@ export function getAchievementImageKey(imageKey: string) {
     return join(ACHIEVEMENT_IMAGE_DEFAULT_PATH, `${imageKey}`);
 }
 
-export function getAchievementImageURL(imageKey: string) {
-    return accessURLForKey(imageKey);
+export async function getAchievementImageURL(template: Achievement_template, state?: achievement_state, achievementContext?: Prisma.JsonValue) {
+    const templateIdsForCourseImage = [10];
```

Comment:
I think this should not be static 🤔 
You could add another type to the [TemplateSelectEnum](https://github.com/corona-school/backend/blob/cb7425895d827b4b756cd3e47cfdac4221f0bbfa/common/achievement/template.ts#L10), like `BY_COURSE_RELATION`.

Reply:
so I took your advice and used the filter and map to get the id's from our templates with course relation.

Reply:
so I took your advice and used the filter and map to get the id's from our templates with course relation.